site/en/docs/extensions/mv3/manifest/activeTab/index.md

```diff
@@ -46,7 +46,7 @@ See the [Page Redder][2] sample extension:
   "browser_action": {
```

Comment:
This should be "action", as we've unified the action types—can you update this?

Confusingly (cc @dotproto) the source doesn't have this JSON at all, I think that's fine because extensions get icons by default, there just won't be text.

Reply:
Yeah sure thing, I will update that to "action".